inverntory_management/products/apps.py

```diff
@@ -0,0 +1,6 @@
+from django.apps import AppConfig
+
+
+class ProductsConfig(AppConfig):
+    default_auto_field = 'django.db.models.BigAutoField'
```

Comment:
I think you can add this into settings file.

Reply:
This file was generated automatically by the Django boilerplate..... Should I keep it this way or add this stuff to setting file? 
Meanwhile I will look into other issues.